chai-as-promised/.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
I found numerous issues with inconsistent indentation in these files, even in the same folders, so this aids in preventing those types of issues in the future. I was requested to remove the .editorconfig file at the root, though, which I did.